stable/airflow/values.yaml

```diff
@@ -174,6 +174,18 @@ airflow:
   ## Run initdb when the scheduler starts.
   initdb: true
 
+  # If set, must be the name of a configmap that contains at least one entry, which key must be named 'run.sh',
+  # and corresponds to a shell script that will be executed in the scheduler container after it has finished
+  # initdb and other standard startup scripts (just before it starts).
+  # Other entries provided in the same configmap will also be mounted in the same directory (using the key name
+  # as their filename). This allows you to split your initialization tasks into several scripts that you can call from
+  # the main 'run.sh' script, for example.
+  # Notes:
+  # - the behaviour of those extra scripts should be idempotent, and fast enough, as they will be executed every time
+  #   the scheduler restarts
+  # - if those scripts use sensitive data, you should provide the sensitive data using secrets
+  #   (see .Values.airflow.extraConfigmapMounts and .Values.airflow.extraEnv)
+  extraStartupScripts:
```

Comment:
Ok, perhaps I'm missing something, but, is there a way to do this with just helm or, does it require having a configmap ran on the cluster prior to running the helm chart?